ckan/ckanext/ckanext-sixodp_showcase/ckanext/sixodp_showcase/cli.py

```diff
@@ -118,15 +117,38 @@ def create_platform_vocabulary(dryrun):
 def migrate_website_showcase_type_to_platform(ctx, dryrun):
     showcase_patches = []
     for old_showcase_dict in package_generator('*:*', 1000, dataset_type='showcase'):
-        if 'showcase_type' not in old_showcase_dict:
-            continue
+        apply_patch = None
+        apply_website = None
+        showcase_extras = []
+        # Look for website in the extras field and append the rest to the showcase extras
+        for item in old_showcase_dict.get('extras', []):
+            if 'showcase_type' in item.get('key'):
+                # Look for website in the showcase_type
+                if 'website' in item.get('value'):
```

Comment:
this might fail in lowercase/uppercase differences

Reply:
Fixed in 0b5b0a7